go-controller/pkg/libovsdbops/meter_test.go

```diff
@@ -1,279 +0,0 @@
-// Unit tests for meter.go.
-// This may be a bit finicky in the future as both the libovsdbclient.Client and libovsdbclient.ConditionalAPI
```

Comment:
The idea of using model_client is that model_client is well tested so that we mostly don't need to test everything else. `meter.go` is heavily based on model_client so I don't see the need for additional testing there just yet. There is though some op logic in libovsdbops that is not fully covered by model_client for which we need better test coverage or enhance model_client to cover those use cases. But nothing that changed much with this PR anyway, we either have basic coverage through unit test of the libovsdbops user code or it was not being unit tested before. So yeah, some work left to do in this regard but I would like to leave it out of the scope of this PR.

Reply:
Actually spotted an issue and consequently added a test. Check it out.